spec/models/person_spec.rb

```diff
@@ -29,6 +27,15 @@
       expect(person.valid?).to be false
     end
   end
+
+  describe "create" do
+    it "generates uuid" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.